### PR TITLE
Feature/gitlab bitbucket/connector

### PR DESCRIPTION
**Pull request by Zarif99**

### CODE REVIEW

File: swagauth/settings.py [hunk 1]
```diff
@@ -152,15 +152,34 @@
 ]
 
 SWAGAUTH_SETTINGS = {
+    'bitbucket': {
+        'APP': {
+            'key': env.str('BITBUCKET_SECRET'),
+            'secret': env.str('BITBUCKET_KEY'),
+        },
+        'SCOPE': {
+
```
Comment (NikBelyaev) [inline swagauth/settings.py]: `SCOPE` should contain `repository`. Also it should be a list, but not dict.

File: swagauth/settings.py [hunk 2]
```diff
@@ -152,15 +152,34 @@
 ]
 
 SWAGAUTH_SETTINGS = {
+    'bitbucket': {
+        'APP': {
+            'key': env.str('BITBUCKET_SECRET'),
```
Comment (NikBelyaev) [inline swagauth/settings.py]: This must be called `client_id` instead of `key`.

Comment (NikBelyaev) [reply swagauth/settings.py]: Edit #1. Or it's a right way to define bitbucket credentials?

File: swagauth/settings.py [hunk 3]
```diff
@@ -152,15 +152,34 @@
...
         ]
     },
+    'gitlab': {
+        'APP': {
+            'client_id': env.str('GITLAB_CLIENT_ID'),
+            'secret': env.str('GITLAB_SECRET'),
+            'key': '',
+        },
+        'SCOPE': [
+
```
Comment (NikBelyaev) [inline swagauth/settings.py]: `SCOPE` should contain `read_repository`

**Review by NikBelyaev — CHANGES_REQUESTED** [review]
Did you test these connectors? Did you receive the tokens?

File: swagauth/settings.py (hunk 2 above)
Comment (Zarif99) [reply swagauth/settings.py]: yes in bitbucket not client_id it has key

File: swagauth/settings.py (hunk 1 above)
Comment (Zarif99) [reply swagauth/settings.py]: oh, yes it`s my mistake

File: swagauth/settings.py (hunk 2 above)
Comment (NikBelyaev) [reply swagauth/settings.py]: Ok

Comment (Zarif99) [reply swagauth/settings.py]: image

**Review by NikBelyaev — APPROVED** [review]

**Review by NikBelyaev — CHANGES_REQUESTED** [review]
**Did you test these connectors? Did you receive the tokens?**

**Review by NikBelyaev — APPROVED** [review]

**Pull request merged**